src/NuGet.Core/NuGet.Packaging/Strings.Designer.cs

```diff
@@ -22,7 +22,7 @@ namespace NuGet.Packaging {
     // with the /str option, or rebuild your VS project.
     [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
     [global::System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
-    internal class Strings {
+    public class Strings {
```

Comment:
I've hit this too: I assume the strings should be internal, but the designer keeps changing them to public. Is there a setting that'll generate internal strings?